tests/test_client_request.py

```diff
@@ -1230,3 +1230,31 @@ def test_loose_cookies_types(loop: Any) -> None:
 def test_gen_default_accept_encoding(has_brotli: Any, expected: Any) -> None:
     with mock.patch("aiohttp.client_reqrep.HAS_BROTLI", has_brotli):
         assert _gen_default_accept_encoding() == expected
+
+
+async def test_certificate_exists(ssl_ctx, client_ssl_ctx) -> None:
+    async def handler(request):
+        return web.Response(text="ok")
+
+    server = web.Server(handler)
+    runner = web.ServerRunner(server)
+    await runner.setup()
+    site = web.TCPSite(runner, ssl_context=ssl_ctx)
+    await site.start()
+
+    async with aiohttp.ClientSession() as session:
```

Comment:
Isn't this what the aiohttp_client fixture is for?
https://docs.aiohttp.org/en/stable/testing.html#pytest